openshift-ansible.spec

```diff
@@ -85,8 +85,8 @@ touch %{buildroot}%{_datadir}/ansible/%{name}/roles/contiv/.empty_dir
 %doc README*
 %license LICENSE
 %dir %{_datadir}/ansible/%{name}
-%{_datadir}/ansible/%{name}/inventory/dynamic
-%ghost %{_datadir}/ansible/%{name}/playbooks/common/openshift-master/library.rpmmoved
```

Comment:
IINM, the `%ghost` is related to the `library.rpmmoved` itself. So the file gets removed when the `openshift-ansible` rpm gets uninstalled (see [1]). I would keep it there (not sure if the `library.rpmmoved` file still gets generated).

[1] http://ftp.rpm.org/max-rpm/s1-rpm-inside-files-list-directives.html#S3-RPM-INSIDE-FLIST-GHOST-DIRECTIVE

Reply:
Not sure why would we want to keep dynamic scripts (for GCP only at this point) when the package is being uninstalled

Reply:
With the `%ghost` directive the file gets removed when the package gets uninstalled. We should keep it that way. Unless that library file is no longer generated. In that case I guess the line can be removed completely.

Reply:
Right, so `library.rpmmoved` is no longer generated there since 3.0+, so its safe to remove this